src/Sulu/Bundle/MediaBundle/Collection/Manager/CollectionManagerInterface.php

```diff
@@ -97,7 +97,7 @@ public function getCount();
      *
      * @return Collection
      */
-    public function save(array $data, int $userId, bool $breadcrumb = false);
+    public function save(array $data, ?int $userId = null, bool $breadcrumb = false);
```

Comment:
As adding `?int` also to the interface is a little bc break I also did add a default value for the userId